File: Hello_Python/day07/mydelete.py
import pymysql

conn = pymysql.connect(host='localhost', user='root', password='python', db='python', charset='utf8', port=3305)


e_id = '5'

cursor = conn.cursor()
sql = f"""
    delete from emp 
    where e_id = '{e_id}' """

print(sql)
cnt = cursor.execute(sql)
print(cnt)

conn.commit()
cursor.close()
conn.close()

File: Hello_Node/mydelete.js
var mysql = require('mysql');
var connection = mysql.createConnection({
	host: 'localhost',
	user: 'root',
	password: 'python',
	database: 'python',
	port: 3305
});

connection.connect();

var e_id = '5';


var sql = `
	delete from emp 
	where
		e_id ='${e_id}'
`;

console.log(sql);

connection.query(sql, function (error, result) {
	console.log("-----------------------------------");
	console.log(error);
	console.log("-----------------------------------");
	console.log(result.affectedRows);

});

connection.end();
